server.js

```diff
@@ -12,7 +11,12 @@ const logger = require('./logger')
 const app = express()
 
 app.config = config
-app.use(cors())
+
+if (process.env.NODE_ENV !== 'production') {
+  const cors = require('cors')
```

Comment:
Using `require` here. So that it won't be called unless is a non-production environment